Makefile

```diff
@@ -75,31 +75,31 @@ update_tools:
 	@go get -u $(GOTOOLS)
 
 tools:
-	@echo "--> Installing tools"
-	@go get $(GOTOOLS)
-	@gometalinter --install
+	echo "--> Installing tools"
+	go get $(GOTOOLS)
+	$(GOPATH)/bin/gometalinter.v2 --install
 
 ### Formatting, linting, and vetting
 
 metalinter:
-	@gometalinter --vendor --deadline=600s --enable-all --disable=lll ./...
+	gometalinter.v2 --vendor --deadline=600s --enable-all --disable=lll ./...
 
-metalinter_test:
-	@gometalinter --vendor --deadline=600s --disable-all  \
+metalinter_test: tools get_vendor_deps
```

Comment:
don't you think the same https://github.com/tendermint/tendermint/pull/902#discussion_r155934459 applies here. I.e. if we want developers actually running this, we want this to be as fast as it can

Reply:
you're right; i was trying to avoid complicating the CI, but it's actually not bad